test/lib/usd/translators/testUsdExportImportRoundtripPreviewSurface.py

```diff
@@ -65,6 +66,10 @@ def __testUsdPreviewSurfaceRoundtrip(self,
         """
         Tests that a usdPreviewSurface exports and imports correctly.
         """
+        mark = Tf.Error.Mark()
+        mark.SetMark()
+        self.assertTrue(mark.IsClean())
```

Comment:
This unit test used to report the "sourceImage" error 10 times. Adding TfMarks makes sure the unit test runs clean and proves the issue is fixed.

Reply:
(would be a good idea to generalize the use of these marks to make sure we never ignore a USD error)

Reply:
Great find! And agreed!